### ISSUE TITLE
Calendar API working on simulator but not published app.

### ISSUE THREAD

**Issue by thenewt15** [expo/expo#1520]
I have an ejected ExpoKit app using SDK v25, and have the calendar API working on my simulator. It asks for permissions, and adds an event to the user's calendar. 

However, I just deployed an AdHoc iOS build to my device, published the javascript with Expo, and the calendar is not working on a real device. No events shows up on my calendar. And I’m not getting back an eventID when I try to create an event (which is how I programmatically determine if it succeeded).

So to reiterate, it works on the simulator when using `exp start` and then running from XCode. But it doesn’t work when I `exp publish` and then distribute a binary. Is there something I’m missing?

**Comment by terribleben** [comment]
Hey there,

- Can you try checking the embedded `Info.plist` of your standalone app and make sure it contains strings for Calendar and Event permissions? If not, there might be an issue on our end.
- Just in case this is a production JS issue, can you try running the app locally but with minify=true and dev=false? That will bring you closer to what you're seeing in the standalone app, though it may not solve the issue.

Lastly, if you can provide a small version of the code you're using in [Snack](https://snack.expo.io) form, that would help us out. I know it's not really possible to test a Snack as a standalone app, but it will still help make sure we're talking about roughly the same code.

**Comment by thenewt15** [comment]
Sorry, I kind of lost hope that anyone was going to reply. I posted this issue 16 days ago and just heard crickets. 
https://forums.expo.io/t/using-new-calendar-api/7346/10?u=thenewt15

I had to move forward so made a new project with `react-native init` and have calendar working now with react-native-calendar-events.

**Comment by terribleben** [comment]
Sorry you were blocked on this. We're a pretty small team and didn't have a chance to look into the issue yet. We'd still like to look into the issue in case others have the same behavior, so I'm going to continue tracking. :)

**Comment by Nikhil-Curefit** [comment]
Hi this issue still exists. Here is the snack
https://snack.expo.io/@drako999/Y3JlYX

**Comment by esamelson** [comment]
Hi, we're closing inactive issues that apply to SDK versions we no longer support. I wasn't able to reproduce this just now on SDK 35. Please create a new issue with a full repro case if this still happens in any of the currently supported SDK versions.
